internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -2266,7 +2282,7 @@ func getVersionKey(openshiftVersion string) (string, error) {
 	return fmt.Sprintf("%d.%d", v.Segments()[0], v.Segments()[1]), nil
 }
 
-func newSecretEnvVar(name, key, secretName string) corev1.EnvVar {
+func newStaticSecretEnvVar(name, key, secretName string) corev1.EnvVar {
```

Comment:
What do you think about leaving the function name as is?

```suggestion
func newSecretEnvVar(annotations map[string]string, name, key, secretName string) corev1.EnvVar {
    secretName = getSecretName(annotations, secretName)
```

and removing the function below https://github.com/openshift/assisted-service/pull/7746/files#diff-c619bb36181b5e1dd76fb1fcc1f54068b16a8761f77106052fcbf5e2f5f9d356R2299-R2302

Reply:
I think renaming the wrapper makes it more intuitive to follow without having to read the function body. What disadvantages do you see in doing this? Do you think naming can be improved?